From: Jakob Schiotz <schiotz@fysik.dtu.dk>
Date: Tue Aug 12 13:41:02 CEST 2025

Remove hack in GPAW's setup.py that may alter $_PYTHON_HOST_PLATFORM in
some cases, as that will make 'pip check' fail.

diff --git a/setup.py b/setup.py
index 839a53a025..caa7c1ab00 100644
--- a/setup.py
+++ b/setup.py
@@ -220,10 +220,6 @@ if parallel_python_interpreter:
 if mpi:
     print('Building GPAW with MPI support.')
 
-platform_id = os.getenv('CPU_ARCH')
-if platform_id:
-    os.environ['_PYTHON_HOST_PLATFORM'] = get_platform() + '-' + platform_id
-
 if gpu:
     valid_gpu_targets = ['cuda', 'hip-amd', 'hip-cuda']
     if gpu_target not in valid_gpu_targets:
